### PR TITLE
Add missing logos for integrations closes #13110

### LINKED ISSUE #13110: Add missing logos for integrations
Hello,

@ecerulm [added](https://github.com/apache/airflow/pull/13073) a logo `field` for `integration` object defined in `provider.yaml`
https://github.com/apache/airflow/blob/ccaca0af3933306c7ca0aa3d78fa2237ff3bfb19/airflow/provider.yaml.schema.json#L45-L48

We then use these logos to promote the integrations on our website.
Screenshot 2020-12-16 at 14 59 59

It would be fantastic if we could add the missing logos to make each integration equally fantastic.

Here is an integration list that does not have a logo. If you are interested, you can integrate only to a few integrations and we will still be happy.
```
$ find . -name provider.yaml | xargs -n 1 yq r --tojson | jq '.integrations | map(select(.logo == null))[] | .["integration-name"]' -r | sort
jq: error (at <stdin>:11): Cannot iterate over null (null)
AWS DataSync
AWS Step Functions
Amazon CloudFormation
Amazon DataSync
Amazon ECS
Amazon ElastiCache
Amazon Glacier
Amazon SecretsManager
Amazon Simple Email Service (SES)
Amazon Web Services
Apache Kylin
Apache Livy
Atlassian Jira
Celery
Databricks
Datadog
Dingding
Discord
Docker
Docker Swarm
Elasticsearch
Exasol
Facebook Ads
File Transfer Protocol (FTP)
Google
Google API Python Client
Google Ads
Google Analytics360
Google BigQuery Data Transfer Service
Google Campaign Manager
Google Cloud
Google Cloud Data Loss Prevention (DLP)
Google Cloud Firestore
Google Cloud Life Sciences
Google Cloud OS Login
Google Cloud Secret Manager
Google Cloud Stackdriver
Google Data Catalog
Google Data Fusion
Google Data Proc
Google Dataprep
Google Deployment Manager
Google Discovery API
Google Display&Video 360
Google Drive
Google Search Ads 360
Google Spreadsheet
Hashicorp Vault
Hypertext Transfer Protocol (HTTP)
IBM Cloudant
Internet Message Access Protocol (IMAP)
Java Database Connectivity (JDBC)
Jenkins
Kubernetes
Microsoft Azure
Microsoft Azure Batch
Microsoft Azure Data Explorer
Microsoft Azure FileShare
Microsoft SQL Server (MSSQL)
MongoDB
MySQL
ODBC
OpenFaaS
Opsgenie
Oracle
Pagerduty
Papermill
Plexus
PostgreSQL
Presto
Qubole
Redis
SQLite
SSH File Transfer Protocol (SFTP)
Salesforce
Samba
Secure Shell (SSH)
Segment
Singularity
Slack
Snowflake
Spark on Kubernetes
Telegram
Vertica
Windows Remote Management (WinRM)
Yandex.Cloud
Yandex.Cloud Dataproc
Zendesk
gRPC
```

CC: @ecerulm

### PR DESCRIPTION
**Pull request by griffincosgrove**
Added missing logos .png images and updated provider.yaml file to point to the path for the logo image. Currently, there are many logos on the Apache Site that do not have logo images.

### CODE REVIEW

**Comment by griffincosgrove** [comment]
@mik-laj addresses issue #13110. Let me know if any further action by me is needed.

File: airflow/providers/apache/livy/provider.yaml [hunk 1]
```diff
@@ -26,6 +26,7 @@ versions:
 integrations:
   - integration-name: Apache Livy
     external-doc-url: https://livy.apache.org/
+    logo: /integration-logos/apache/livy.png
```
Comment (mik-laj) [inline airflow/providers/apache/livy/provider.yaml]:
```suggestion
    logo: /integration-logos/apache/Livy.png
```

File: airflow/providers/google/provider.yaml [hunk 2]
```diff
@@ -182,11 +191,13 @@ integrations:
     tags: [gcp]
   - integration-name: Google Data Proc
     external-doc-url: https://cloud.yandex.com/services/data-proc
+    logo: /integration-logos/gcp/Google-Data-Proc.png
     tags: [gcp]
   - integration-name: Google Data Catalog
     external-doc-url: https://cloud.google.com/data-catalog
     how-to-guide:
       - /docs/apache-airflow-providers-google/operators/cloud/datacatalog.rst
+    logo: /integration-logos/gcp/Google-Cloud-Data-Catalog.png
```
Comment (mik-laj) [inline airflow/providers/google/provider.yaml]:
```suggestion
    logo: /integration-logos/gcp/Google-Data-Catalog.png
```

File: airflow/providers/google/provider.yaml [hunk 3]
```diff
@@ -74,11 +77,13 @@ integrations:
     external-doc-url: https://cloud.google.com/dlp/
     how-to-guide:
       - /docs/apache-airflow-providers-google/operators/cloud/data_loss_prevention.rst
+    logo: /integration-logos/gcp/Google-Data-Loss-Prevention.png
```
Comment (mik-laj) [inline airflow/providers/google/provider.yaml]:
```suggestion
    logo: /integration-logos/gcp/google-data-loss-prevention.png
```

File: airflow/providers/google/provider.yaml [hunk 4]
```diff
@@ -216,41 +229,51 @@ integrations:
...
     tags: [gcp]
   - integration-name: Google Discovery API
     external-doc-url: https://developers.google.com/discovery
+    logo: /integration-logos/gcp/Google-Cloud-Generic.png
     tags: [google]
   - integration-name: Google Display&Video 360
     external-doc-url: https://marketingplatform.google.com/about/display-video-360/
     how-to-guide:
       - /docs/apache-airflow-providers-google/operators/marketing_platform/display_video.rst
+    logo: /integration-logos/gcp/Google-Display-And-Video360.png
```
Comment (mik-laj) [inline airflow/providers/google/provider.yaml]:
```suggestion
    logo: /integration-logos/gcp/Google-Display-And-Video-360.png
```

**Comment by mik-laj** [comment]
Pre-commit detected a few path issues so I fixed them.

**Comment by mik-laj** [comment]
Most look fine, but I have a reservation about the few logos that have chess in the background. 
Screenshot 2021-01-11 at 11 05 53
Screenshot 2021-01-11 at 11 05 58
Screenshot 2021-01-11 at 11 06 04
Screenshot 2021-01-11 at 11 07 02
Screenshot 2021-01-11 at 11 07 08
Screenshot 2021-01-11 at 11 07 33
As for the AWS icon, I think it should contain an orange color.
Screenshot 2021-01-11 at 11 07 24
What do you think about https://seeklogo.com/vector-logo/319188/amazon-web-services-aws ?

**Comment by potiuk** [comment]
> Most look fine, but I have a reservation about the few logos that have chess in the background.

Likely badly exported images. Chess usually indicates transparency, so the logos used to have transparency, but they were probably copy&pasted rather than exported from GiMP or similar.

**Comment by griffincosgrove** [comment]
@potiuk i believe you are correct with that logic. I think for for some logos I didn't export with Gimp. I can export those and hopefully it will fix the "chess" issue.

@mik-laj i agree with the AWS logo. I can change that out as well. 

Thanks for the feedback I can make these changes.

**Comment by mik-laj** [comment]
@griffincosgrove  I can still see some logos have chess backgrounds.
Screenshot 2021-01-13 at 12 53 10
Screenshot 2021-01-13 at 12 57 43
Screenshot 2021-01-13 at 12 57 51
Screenshot 2021-01-13 at 12 57 57
Screenshot 2021-01-13 at 12 58 18
Screenshot 2021-01-13 at 12 58 36
Screenshot 2021-01-13 at 12 58 44

I noticed one more problem. The Dataproc logo has a margin, but the other logos for GCP services have no additional margin. It looks a bit weird when several GCP services are side by side.
Screenshot 2021-01-13 at 12 53 17

**Comment by griffincosgrove** [comment]
@mik-laj Ok looks like exporting them did not fix the issue. I'll look into alternative options. 
I had some difficulty with the GCP icons but I'll make that one bigger.

**Comment by mik-laj** [comment]
@griffincosgrove  It looks like Google has updated its icons because previously you could download diamond-shaped icons from the GCP website.
https://cloud.google.com/icons
You can also download icons from https://vecta.io/symbols/4/google-cloud-platform

**Comment by griffincosgrove** [comment]
@mik-laj thanks! I downloaded and used the png/svg icons zip where appropriate. I will replace the G Data Proc icon and search for new ones for those checker icons.

**Comment by griffincosgrove** [comment]
@mik-laj replaced the checkers logos and the Google Data Proc logo 😄

**Comment by potiuk** [comment]
Hello @griffincosgrove -> please rebase again :( - there was a problem I fixed in master just now

**Comment by griffincosgrove** [comment]
thought i followed the instructions on rebasing but I feel like I messed something up. Did i mess something up @potiuk & @mik-laj

**Comment by mik-laj** [comment]
It looks like something is wrong. We didn't get any change. Could you please try to restore the previous versions?  I'll do a rebase and it should work.

I'm not sure, but I guess you could have put the main branch together and then rebase it. This can cause problems because Git cannot find the correct commits.

**Comment by mik-laj** [comment]
You can also create a new PR if you manage to recover the files.

**Pull request closed without merging**